GitUI/NBugReports/BugReporter.cs

```diff
@@ -46,85 +47,73 @@ private static string FormatText(ExternalOperationException ex, bool canRaiseBug
         }
 
-        private static void ReportException(string text, string instructionText, Exception ex, bool isTerminating)
-        {
-            using var dialog = new TaskDialog
-            {
-                OwnerWindowHandle = OwnerFormHandle,
-                Text = text,
-                InstructionText = instructionText,
-                Caption = Strings.Error,
-                Icon = TaskDialogStandardIcon.Error,
-                Cancelable = true,
-            };
-            var btnReport = new TaskDialogCommandLink("Report", Strings.ButtonReportBug);
-            btnReport.Click += (s, e) =>
-            {
-                dialog.Close();
-                ShowNBug(OwnerForm, ex, isTerminating);
-            };
-            var btnIgnoreOrClose = new TaskDialogCommandLink("IgnoreOrClose", isTerminating ? Strings.ButtonCloseApp : Strings.ButtonIgnore);
-            btnIgnoreOrClose.Click += (s, e) =>
-            {
-                dialog.Close();
-            };
-            dialog.Controls.Add(btnReport);
-            dialog.Controls.Add(btnIgnoreOrClose);
-
-            dialog.Show();
-        }
-
-        private static void ReportGenericException(Exception ex, bool isTerminating)
+        private static void ReportGenericException(Exception exception, bool isTerminating)
         {
             // This exception is arbitrary, see if there's additional information
-            string? moreInfo = ex.InnerException?.Message;
+            string? moreInfo = exception.InnerException?.Message;
             if (moreInfo != null)
             {
                 moreInfo += Environment.NewLine + Environment.NewLine;
             }
 
-            ReportException($"{moreInfo}{Strings.ReportBug}", ex.Message, ex, isTerminating);
+            ShowException($"{moreInfo}{Strings.ReportBug}", exception.Message, exception, isTerminating);
         }
 
-        private static void ReportUserException(UserExternalOperationException ex, bool isTerminating)
+        private static void ReportUserException(UserExternalOperationException exception, bool isTerminating)
+            => ShowException(FormatText(exception, canRaiseBug: false), exception.Context, exception: null, isTerminating);
+
+        private static void ShowException(string text, string instructionText, Exception? exception, bool isTerminating)
         {
             using var dialog = new TaskDialog
             {
                 OwnerWindowHandle = OwnerFormHandle,
-                Text = FormatText(ex, canRaiseBug: false),
-                InstructionText = ex.Context,
-                Caption = Strings.CaptionFailedExecute,
+                Text = text,
+                InstructionText = instructionText,
+                Caption = Strings.Error,
                 Icon = TaskDialogStandardIcon.Error,
                 Cancelable = true,
             };
+
+            if (!(exception is null))
```

Comment:
Nit:
```suggestion
            if (exception != null)
```
will be easier to replace when enabling C#9 features.